### ISSUE TITLE
[Roadmap] Naming of conversations

### ISSUE THREAD

**Issue by david-strejc** [enricoros/big-AGI#297]
Implement custom naming of conversations. 

I got lot of conversations going on and I want to be able to name them.

**Comment by enricoros** [comment]
Approved.

**Comment by enricoros** [comment]
Ideally depends on #201 #162 #296.

Note: have an idea on how to implement this:

- in the chat lists (which will be converted to a pane on Desktop and Drawe on mobile)
  - [ ] have a pencil button to edit the title
  - [ ] have a  'magic' button to re-auto-title the conversation

**Comment by enricoros** [comment]
Also, duplicate of #222

**Comment by enricoros** [comment]
@david-strejc with the 1.10.0 released today, you can double-click on any conversation to rename them.
The "Edit button" and "Auto-rename" (dice?) button will come soon.

**Comment by enricoros** [comment]
![Image](https://github.com/enricoros/big-AGI/assets/32999/db78455c-c15a-42f0-9577-f265fbd2da54)

@david-strejc Completed, includes manual editing of the title, or auto naming.
